### ISSUE TITLE
Some homepage icons can get inappropriately cropped

### ISSUE THREAD

**Issue by erogray** [GetDKAN/dkan#2820]
It looks like some homepage icons don't quite fit into the space allotted to them, and consequently have an unattractive cropped appearance.
<img width="351" alt="screen shot 2019-02-12 at 9 56 43 am" src="https://user-images.githubusercontent.com/21176376/52657411-7a71a980-2ead-11e9-9431-aadb57c3779b.png">
<img width="388" alt="screen shot 2019-02-12 at 10 01 29 am" src="https://user-images.githubusercontent.com/21176376/52657412-7b0a4000-2ead-11e9-955e-08dcd5eaf81c.png">
<img width="183" alt="screen shot 2019-02-12 at 10 02 08 am" src="https://user-images.githubusercontent.com/21176376/52657413-7b0a4000-2ead-11e9-9a3a-613a6135e1c7.png">

**Comment by dharizza** [comment]
I've tried changing the units-per-em, adding `display: inline-block` and `display: block` to the <a> that has the `::before` with the icon, also tried to change the overflow, the opacity, added `text-rendering:optimizeLegibility;` and `-webkit-font-feature-settings: "kern" 1;`. Changed the order of the elements in dkan-topics.css to load svg first, then woff and finally ttf, but had no success. I also tried changing the font-size units to px and vw but no success, the only moment in which it seems to work is when we increase the font-size from 7em to 17em, but that's way too big.
Invoking @janette 's magic.
